.github/actions/pull_request_semver_label_checker.yml

```diff
@@ -0,0 +1,9 @@
+name: 'Pull request semver label checker'
+description: 'Checks that at least one semver label is applied to the pull request'
+runs:
+    using: "composite"
+    steps:
+      - name: Check labels
+        run: |
+            gh pr view 2809 --repo apple/swift-nio --json labels \
```

Comment:
Don't we want to get the current PR number or is this just hardcoded for testing and to be updated in a later PR?